src/Symfony/Component/Cache/Adapter/ArrayAdapter.php

```diff
@@ -177,4 +172,16 @@ private function validateKey($key)
 
         return $key;
     }
+
+    private function createGenerator(array $keys)
```

Comment:
this is still not inlining the function. It still uses a separate function, even though it is anonymous rather than named.